src/controller.cpp

```diff
@@ -561,8 +561,8 @@ int Controller::run(const CliArgsParser& args)
 		if (!args.silent()) {
 			std::cout << _("done.") << std::endl;
 			if (amt > 0u) {
-				std::cout << _("Unreachable feeds found, consider setting "
-						"`cleanup-on-quit yes` or run `newsboat --cleanup`")
+				std::cout << strprintf::fmt("%d unreachable feeds found. See "
+						"`cleanup-on-quit` in newsboat(1) for details.", amt)
```

Comment:
```suggestion
				std::cout << strprintf::fmt(_("%d unreachable feeds found. See "
						"`cleanup-on-quit` in newsboat(1) for details."), amt)
```

Translatable strings have to be annotated with `_()`.

Reply:
Fixed.